src/types/env.ts

```diff
@@ -55,9 +56,21 @@ const mtProtoSettings = T.Object({
   TELEGRAM_API_HASH: T.String(),
 });
 
+const storageSettings = T.Object({
+  /**
+   * The supabase instance url for storing chats, sessions, etc.
+   */
+  SUPABASE_URL: T.String(),
+  /**
+   * The supabase service key for storing chats, sessions, etc.
+   */
+  SUPABASE_SERVICE_KEY: T.String(),
```

Comment:
Service key is new lingo. Shouldn't it just be `SUPABASE_KEY`?

Reply:
It's supposed to be the literal `service_key` not the `anon_key`, I felt blocking everyone from reading the tables was best but probs not required esp with GitHub storage in a week or two.

Reply:
I think it makes sense to retain what you did and then change our anon key secret name to actually include the anon part.

Reply:
This key is embedded inside a json string for this plugin so it won't have immediate access unless I move it into global scope.

Also I'm using my supabase details for this plugin right now too but good idea in general